### ISSUE TITLE
Review sign-up redirect pathway

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#341]
**TL;DR**: Make sure it's clear to a new participant how to get started.

**Narrative**
"As a participant, online studies are new to me and I may expect that after signing up I need to wait to hear from a human that I am invited to participate in something. I want it to be clear how to proceed from making an account to participating in a study."

**Acceptance Criteria**

- [ ] Pathway from account creation through to study reviewed and language added as needed to guide new participant through during initial setup, both from single-study page and from homepage
- [ ] Observe at least a few naive participants create an account and participate, e.g. at museum
- [ ] Sign off from Kim

**Implementation Notes**
